src/corrgi/corrgi.py

```diff
@@ -21,13 +22,16 @@ def compute_autocorrelation(
         catalog (Catalog): The catalog.
         random (Catalog): A random samples catalog.
         params (Munch): The parameters dictionary to run gundam with.
+        use_weights (bool): Whether to use weights or not. Defaults to False.
 
     Returns:
         A numpy array with the result of the auto-correlation, using the natural estimator.
     """
+    if use_weights and "wei" not in catalog.columns:
```

Comment:
Could there be a parameter for the weight_column name?

Reply:
Great idea, thank you!